RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -356,21 +376,7 @@ private extension String {
 
 }
 
-protocol Transaction {
-
-    var productIdentifier: String { get }
-    var store: Store { get }
-    var type: TransactionType { get }
-    var isCancelled: Bool { get }
-}
-
-enum TransactionType {
-
-    case subscription(isActive: Bool, willRenew: Bool, expiresDate: Date?, isTrial: Bool)
-    case nonSubscription
-}
-
-extension RevenueCat.SubscriptionInfo: Transaction {
+@_spi(Internal) extension RevenueCat.SubscriptionInfo: Transaction {
```

Comment:
what does spi internal give us here?

Reply:
`managementURL` is under the `@_spi` scope, so we need it in order to access it